### DISCUSSION TITLE
Automatic zoom on alert

### DISCUSSION THREAD

**Discussion by danieldegroot2** [xspanger3770/GlobalQuake#120] [Ideas]
Idea by Soba on Discord;
When an earthquake occurs, automaticly zoom in to the relevant area.

![image](https://github.com/xspanger3770/GlobalQuake/assets/67521919/c8291be4-726c-4344-ba48-31996f69cbec)

**Comment by xspanger3770** [comment]
Yes, this is very often requested feature, I will create an issue...

**Comment by danieldegroot2** [comment]
Superseeded by https://github.com/xspanger3770/GlobalQuake/issues/121
